### ISSUE TITLE
Display the default value of args in help

### ISSUE THREAD

**Issue by yahya-abou-imran** [nat-n/poethepoet#126]
Thanks for this absolutely wonderful lib!

Would it be possible to display the default value of an args in the help?

Example:

```toml
    [tool.poe.tasks.load]
    help = "Load base data"
    control.expr = "pack"

        [[tool.poe.tasks.load.args]]
        name = "pack"
        options = ["-p", "--pack"]
        help = "The amount of fixtures needed"
        default = "full"

        ...
```

Now the help displays:

```shell
  load           Load base data
    -p, --pack   The amount of fixtures needed
```

It would be useful to add this:

```shell
    -p, --pack   The amount of fixtures needed [Default: full]
```

I could write it by hand, but using directly the default is more DRY.

Thanks!

**Comment by nat-n** [comment]
Hi @yahya-abou-imran, Thanks for the feedback :)

I think this is a good idea. I'd like to overhaul how documentation works at some point to make it more detailed, but this is a step in the right direction. Feel free to submit a PR, otherwise I'll add it when I next have some time.

**Comment by nat-n** [comment]
@yahya-abou-imran This was shipped in [0.19.0](https://pypi.org/project/poethepoet/0.19.0/) 🙂
